charters/DataOperations/charter.md

```diff
@@ -0,0 +1,89 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
+
+## Definitions [Optional]
+
+Data submission: a set of raw data derived from the same experimental process, and the metadata describing that process
+Data release: the point at which metadata and data from a single data submission become publicly available
+Data distribution: the packaging of metadata and data from multiple data submissions in meaningful ways and any announcements or documentation supporting it
+
+## Objectives
+
+The objective of the Data Operations is to provide the scientific community with effective tools and resources from the DCP which maximize the value and use of HCA data
```

Comment:
Related to Ambrose's suggestion above: I'd make these objectives as concrete as possible.

Reply:
@mweiden does 0de0c59 address your concern?

Reply:
yup, thanks